staging/cse/windows/provisioningscripts/kubeproxystart.ps1

```diff
@@ -33,8 +70,23 @@ if ($KubeproxyFeatureGates -contains "WinDSR=true") {
     $global:KubeproxyArgList += @("--enable-dsr=true")
 }
 
-if ($KubeproxyFeatureGates.Count -ne 0) {
-    $global:KubeproxyArgList += @("--feature-gates=" + ($KubeproxyFeatureGates -join ","))
+$featureGateArgs = ""
+foreach ($feature in $KubeproxyFeatureGates) {
+    # IPv6DualStack feature gate should not be passed to kube-proxy in >= 1.25.0
+    # https://github.com/kubernetes/kubernetes/blob/ef70d260f3d036fc22b30538576bbf6b36329995/pkg/features/kube_features.go#L945
+    if (($feature -like "IPv6DualStack=*") -and ((Compare-SemanticVersion -Version1 $KubernetesVersion -Version2 "1.25.0") -ge 0)) {
```

Comment:
to be honest we may not need the sem ver check since IPv6DualStack has been enabled by default since 1.21.